dtfs-central-api/src/v1/controllers/tfm/deal/tfm-submit-deal.controller.js

```diff
@@ -128,7 +128,7 @@ exports.submitDealPut = async (req, res) => {
   }
 
   try {
-    validateAuditDetailsAndUserType(auditDetails, 'portal');
+    validateAuditDetailsAndUserType(auditDetails, 'tfm');
   } catch (error) {
```

Comment:
This is used by both tfm and portal

Reply:
Where is it used by a tfm user? I can see its in tfm-api but I think that's called by portal-api